astcontext/funcs_test.go

```diff
@@ -124,9 +124,6 @@ func (q *qaz) example(x,y,z int) error {
 	_ = func(foo int) error {
 		return nil
 	}
-	_ = func() (err error) {
-		return nil
-	}
```

Comment:
Can you please put this back?

Reply:
Those three were added by me [here](https://github.com/fatih/motion/commit/1e09f6b8ad3567c0a06deed9e28ca0fb3ab8de55#diff-8b82e3d489f4b315c5856fc30602bfcb), so I moved them into `TestFunc_Signature_Named_Results` in order to leave original `TestFunc_Signature` as it was before ﻿1e09f6b. Are you sure you want me to move them back?

Reply:
Ok, I've put them back. Can you please check? If it's ok I can squash last two test related commits into one.